plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go

```diff
@@ -22,24 +22,28 @@ The 'request_account_id' and 'request_region' columns are added to show the acco
 		Transform: transformers.TransformWithStruct(&types.StackSetOperationResultSummary{}),
 		Columns: []schema.Column{
 			{
-				Name:     "request_account_id",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: client.ResolveAWSAccount,
+				Name:       "request_account_id",
```

Comment:
I think we are missing a PK field for this table `aws_cloudformation_stack_set_operation_results` I don't see any field from API response in the PK, and the API returns a paginated list of items

Reply:
9b563cdbe204d62fa560f1c49ec32085297f9393